src/ui/AccountStatus.svelte

```diff
@@ -1,21 +1,55 @@
 <script>
-  import Svg from '@/ui/Svg/svelte'
-  import { getSanbaseSubscription, getTrialDaysLeft } from '@/utils/subscription'
+  import { customerData$ } from '@/stores/user'
+  import { subscription$ } from '@/stores/subscription'
+  import { PlanName } from '@/utils/plans'
 
   export let currentUser
+  export let variant = 0
 
-  $: subscription = currentUser && getSanbaseSubscription(currentUser.subscriptions)
-  $: trialDaysLeft = subscription && getTrialDaysLeft(subscription)
-  $: trialPostfix = trialDaysLeft ? ` Trial (${trialDaysLeft})` : ''
+  $: ({ isEligibleForTrial, annualDiscount } = $customerData$)
+  $: trialSpecialOfferPercent =
+    !annualDiscount || !annualDiscount.isEligible || !annualDiscount.discount
+      ? 0
+      : annualDiscount.discount.percentOff
+  $: subscription = $subscription$
+  $: currentPlanName = subscription
+    ? PlanName[subscription.plan.name] || subscription.plan.name
+    : null
 </script>
 
 {#if currentUser}
-  {#if subscription}
-    <a
-      href="https://academy.santiment.net/products-and-plans/sanbase-pro-features/"
-      class="pro btn-1 btn--s caption row v-center">
-      <Svg id="crown" w="13" h="10" class="mrg-s mrg--r" />
-      Pro{trialPostfix}</a>
+  {#if currentPlanName && currentPlanName.toString().toUpperCase().includes('PRO')}
```

Comment:
'currentPlanName &&' eliminates falsy case
```suggestion
  {#if currentPlanName && currentPlanName.toUpperCase().includes('PRO')}
```